src/js/apps/patients/schedule/reduced_schedule_app.js

```diff
@@ -4,7 +4,7 @@ import App from 'js/base/app';
 
 import { STATE_STATUS } from 'js/static';
 
-import SearchComponent from 'js/views/patients/shared/components/list-search';
+import SearchComponent from 'js/views/shared/components/list-search';
```

Comment:
Since the `SearchComponent` is now being used by the both the patients worklist/schedule and clinicians apps, I moved its files to the `/js/views/shared/components` directory.

This import path was also changed for the `schedule_app.js` and `worklist_app.js` files.

Reply:
Yeah good call. I suspect there's probably i18n strings in some of the files that were moved that also need to be moved

Reply:
Yes you're right, there is a i18n string that needed to be updated. Will be included in the next commit.